packages/components/src/components/DistributionChart.tsx

```diff
@@ -22,13 +22,15 @@ export const DistributionChart: React.FC<DistributionChartProps> = ({
 }) => {
   return (
     <MultiDistributionChart
-      plot={{
-        distributions: [
-          { name: "default", distribution: distribution, opacity: 1 },
-        ],
-        showLegend: false,
-        colorScheme: "blues",
-      }}
+      plot={
+        {
+          distributions: [
+            { name: "default", distribution: distribution, opacity: 1 },
+          ],
+          showLegend: false,
+          colorScheme: "blues",
+        } as unknown as SqPlot
```

Comment:
I'm a bit annoyed with this, but I want to create a SqPlot with exactly the same fields, but I don't have access to its constructor. One possible solution to this would be to have the SqPlot object have a `getSpec` method or something that returns the entire spec in a dictionary and we use that as a prop.